### ISSUE TITLE
Unable to install howdy for Ubuntu 22.04 due to *.bz2 files.

### ISSUE THREAD

**Issue by aahlad2000** [boltgolt/howdy#740]
Facing Issues for installation of Howdy for Ubuntu 22.04. Apparently error message says it can't read .bz2 files.
These are the following commands I have used :
**sudo add-apt-repository ppa:boltgolt/howdy
sudo apt update
sudo apt install howdy**

![howdy-error](https://user-images.githubusercontent.com/62693668/210126838-6b3f7a16-ccd5-485f-bba3-b7467f94332c.png)

**Comment by Batwam** [comment]
Could this be a duplicate of this issue? https://github.com/boltgolt/howdy/issues/738

**Comment by Batwam** [comment]
for info, I have run the [dlib-data/install.sh](https://github.com/boltgolt/howdy/blob/beta/howdy/src/dlib-data/install.sh) without problem:
![image](https://user-images.githubusercontent.com/56710655/210210905-56ec3836-f9a4-4c2e-83f8-556859e67a2c.png)

Does it run for you?

**Comment by aahlad2000** [comment]
Unable to run this command.
Screenshot from 2023-01-07 16-49-38.png

**Comment by aahlad2000** [comment]
Unable to run this command
![Uploading Screenshot from 2023-01-07 16-49-38.png…]()

**Comment by aahlad2000** [comment]
Unable to run this command
![Uploading Screenshot from 2023-01-07 16-49-38.png…]()

**Comment by aahlad2000** [comment]
Unable to run this command

file:///home/aahlad_kethineedi/Pictures/Screenshot%20from%202023-01-07%2016-49-38.png

**Comment by aahlad2000** [comment]
Unable to run this command.
![Screenshot from 2023-01-07 16-49-38](https://user-images.githubusercontent.com/62693668/211147786-c9150b08-59c8-4b19-833b-dd4c283dfac5.png)

**Comment by Batwam** [comment]
Yeah, I didn't meant type it as a command, it was just a link to the file. I meant download the script and run it.

**Comment by aahlad2000** [comment]
Still unable to run it.
![ERROR](https://user-images.githubusercontent.com/62693668/211209463-c7193f41-e2ee-490a-961e-402f5f9c619a.jpg)

**Comment by Batwam** [comment]
This is pretty odd as the script is pretty simple and uses wget to download 3 files. It even checks if wget is installed and falls back on `curl` of `wget` isn't installed. I though perhaps you are somehow missing `wget`  or `curl` or even `hash`. You should be seing some sort or error...

Are you able to do the manual download of the 3 files? 
`wget https://github.com/davisking/dlib-models/raw/master/dlib_face_recognition_resnet_model_v1.dat.bz2`
`wget https://github.com/davisking/dlib-models/raw/master/mmod_human_face_detector.dat.bz2`
`wget https://github.com/davisking/dlib-models/raw/master/shape_predictor_5_face_landmarks.dat.bz2`

If so you could just download them, leave them there (don't extract) and run the main code again. This way it should see them and extract when it gets to the extracting phase. I'm sure this is going to solve the main issue as there could be other issues in the code but it's worth a try...

**Comment by aahlad2000** [comment]
I see.
However I am unable to download the 3 files. I got this error :
failed: Network is unreachable.

**Comment by aahlad2000** [comment]
Also, looks like I am unable to access these URLs from the browser as well.

**Comment by Batwam** [comment]
You could give it a try. I had a look and you would need to save the files in `/usr/lib/security/howdy/dlib-data`. 

I don't know how you manage to get a different ppa but using the same commands. The ppa for this package you send you to `http://ppa.launchpad.net/boltgolt/howdy/ubuntu/dists/j` and `jammy is in the Release list.

Note that I also just tried the ppa installation method on a fresh 22.10 install in a VM and it's spitting out quite a few warnings...
![image](https://user-images.githubusercontent.com/56710655/213901250-d3c43bd7-92ad-4225-bc1e-68a8bd7bdb69.png)

**Comment by 3gsahil** [comment]
> This is pretty odd as the script is pretty simple and uses wget to download 3 files. It even checks if wget is installed and falls back on `curl` of `wget` isn't installed. I though perhaps you are somehow missing `wget` or `curl` or even `hash`. You should be seing some sort or error...
> 
> Are you able to do the manual download of the 3 files? `wget https://github.com/davisking/dlib-models/raw/master/dlib_face_recognition_resnet_model_v1.dat.bz2` `wget https://github.com/davisking/dlib-models/raw/master/mmod_human_face_detector.dat.bz2` `wget https://github.com/davisking/dlib-models/raw/master/shape_predictor_5_face_landmarks.dat.bz2`
> 
> If so you could just download them, leave them there (don't extract) and run the main code again. This way it should see them and extract when it gets to the extracting phase. I'm sure this is going to solve the main issue as there could be other issues in the code but it's worth a try...

unable to download the files altogether .

**Comment by Batwam** [comment]
Feels like some sort of proxy issue as the files are available at the locations provided as far as I can tell. Are you using some sort of school/work/public network which may be blocking file storage perhaps?

**Comment by EricMuChR** [comment]
Same problem for me. Also Ubuntu 22.04. Just couldn't find a way to install those 3 files. Is there any hope that this problem will be resolved if I update the system version to like Ubuntu 24.04?
